mkdocs.yml

```diff
@@ -44,6 +44,19 @@ nav:
       - Node Types: ./canarynet/node/node-type.md
       - Run Full Node: ./canarynet/node/run-full-node.md
       - Join as Validator: ./canarynet/node/run-validator.md
+  - NFTFair:
+    - Introduction: ./nftfair/introduction.md
+    - User Info:
```

Comment:
user info? user guide?

Reply:
hhh

Reply:
Well technically, the section doesn't really provide any sort of "guide" for the users, but really presents more information about the platform.